src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -152,6 +153,7 @@ void fillInnerParts() {
         if (clients.size() > 0) {
```

Comment:
I assume when we are first starting the application, we automatically show the Procedures of the first Client. 
Is there a reason why you set it that way, as opposed to just showing a blank screen on the right side (since the user hasn't called listProc yet)?

Reply:
it gives a rough overview of what the right column is for. I think it would just be more intuitive, so if you think it's a bad idea I can remove it!